miner/miner_test.go

```diff
@@ -85,8 +89,26 @@ func TestVerifyProofOfWork(t *testing.T) {
 	b.Target = blockchain.BigIntToHash(
 		c.MaxUint256,
 	)
+	m := New()
 
-	if !VerifyProofOfWork(b) {
-		t.Fail()
-	}
+	assert.True(t, m.VerifyProofOfWork(b))
+}
+
+func TestStopPauseMining(t *testing.T) {
+	b := blockchain.NewTestBlock()
+	b.Target = blockchain.BigIntToHash(constants.MinTarget)
+	m := New()
+
+	go m.Mine(b)
+	time.Sleep(time.Second / 2)
```

Comment:
Just do `time.Millisecond * 50` - this type of stuff really slows our test suite down.